build.xml

```diff
@@ -119,8 +119,14 @@
 	</target>
 	
 	 <target name="test-compile" depends="compile">
-	    <mkdir dir="${test.build.dir}"/>
-	    <javac srcdir="${test.src.dir}" destdir="${test.build.dir}" includeantruntime="false">
+			<mkdir dir="${test.build.dir}"/>
```

Comment:
Can you fix the indentation of this section please?
(And while you're at it maybe the following section as well.)

Reply:
That should be all fixed now, and the rest of the file looks okay.

Reply:
The complete block from <mkdir ... /> to </javac> (ll.122-133) is still indented by one tab too many. The line above it has an erroneous space in front of the tag, which may be throwing your editor off.

Reply:
Is this all fine now?